task-sdk/tests/task_sdk/execution_time/test_task_runner.py

```diff
@@ -1795,7 +1795,9 @@ def execute(self, context):
         spy_agency.assert_spy_called(_push_xcom_if_needed)
 
         if should_push_xcom:
-            spy_agency.assert_spy_called_with(_xcom_push, runtime_ti, "return_value", expected_xcom_value)
+            spy_agency.assert_spy_called_with(
+                _xcom_push, runtime_ti, BaseXCom.XCOM_RETURN_KEY, expected_xcom_value
```

Comment:
I almost wonder if this one is better as the literal string? I don't know it really matters though